mava/utils/make_env.py

```diff
@@ -124,6 +126,41 @@ def make_jaxmarl_env(
     return env, eval_env
 
 
+def make_gigastep_env(
+    env_name: str, config: DictConfig, add_global_state: bool = False
+) -> Tuple[Environment, Environment]:
+    """
+     Create a GIGASTEP environment.
+
+    Args:
+        env_name (str): The name of the environment to create.
+        config (Dict): The configuration of the environment.
+        add_global_state (bool): Whether to add the global state to the observation. Default False.
+
+    Returns:
+        A tuple of the environments.
+    """
+    kwargs = dict(config.env.kwargs)
+
+    scenario_config = dict(config.env.scenario)
+    scenario = ScenarioBuilder.from_config(scenario_config)
+
+    train_env = GigastepWrapper(
+        scenario.make(discrete_actions=True, obs_type="vector"),
```

Comment:
Would be interesting to have an option of making this a continuous env, but we can leave that for a later PR